File: gol_java/src/Coord.java
package src;

public class Coord {
    private byte x = 0;
    private byte y = 0;

    public Coord(byte x, byte y){
        this.x = x;
        this.y = y;
    }

    public byte _x(){
        return this.x;
    }
    
    public byte _y(){
        return this.y;
    }
}

File: maze_solver_a_star/Coord.py
class Coord:

    def __init__(self, X: int, Y: int):
        self.coord = (X, Y)

    @property
    def X(self):
        return self.coord[0]

    @property
    def Y(self):
        return self.coord[1]

    def into_list(self, coord_list: list):
        for coord in coord_list:
            if coord.X == self.X and coord.Y == self.Y:
                return True
        return False
